packages/ui/src/views/layouts/PullRequestLayout.tsx

```diff
@@ -43,55 +65,85 @@ export const PullRequestLayout: FC<PullRequestLayoutProps> = ({
   repoId,
   updateTitle
 }) => {
-  const { Outlet, navigate } = useRouterContext()
+  const { Outlet } = useRouterContext()
   const { pullRequest } = usePullRequestStore()
   const { t } = useTranslationStore()
   const { isInset } = useTheme()
 
-  const getTabProps = useCallback(
-    (tab: PullRequestTabsKeys): TabsTriggerProps => ({
-      value: tab,
-      onClick: () => navigate(`../${tab}`),
-      className: 'group gap-x-1.5',
-      role: 'link'
-    }),
-    [navigate]
-  )
-
   return (
     <SandboxLayout.Main fullWidth>
       <SandboxLayout.Content className="mx-auto max-w-[1500px] px-6">
         {pullRequest && (
           <PullRequestHeader className="mb-10" updateTitle={updateTitle} data={{ ...pullRequest, spaceId, repoId }} />
         )}
 
-        <Tabs.Root variant="tabnav" className="mb-7" defaultValue={PullRequestTabsKeys.CONVERSATION}>
+        <nav
+          className={cn(
+            'relative before:bottom-0 before:absolute before:w-[calc(100vw-var(--sidebar-width))] before:min-w-[calc(100%+3rem)] before:left-1/2 before:-translate-x-1/2 before:border-b before:border-borders-4',
+            { 'before:w-[calc(100vw-var(--sidebar-width)-6px)]': isInset }
+          )}
+        >
+          <ul className="mb-7 flex items-center">
```

Comment:
we should be using the TabNav component instead of html elements with custom styles on it

Reply:
Fixed